modules/dkan/open_data_federal_extras/open_data_federal_extras.install

```diff
@@ -45,3 +45,22 @@ function open_data_federal_extras_update_7002() {
           SET field_odfe_bureau_code_value = REPLACE(field_odfe_bureau_code_value, '-', ':')";
   $result = db_query($sql);
 }
+
+/**
+ * Delete fields when uninstall module.
+ */
+function open_data_federal_extras_uninstall() {
```

Comment:
@jacintocapote this works great, except that the admin menu still shows those fields as existing after the uninstall. Can you add a menu cache rebuild to the uninstall hook and then we should be golden?